data/json/items/ranged.json

```diff
@@ -4213,6 +4213,38 @@
     "valid_mod_locations": [[ "accessories", 4 ],[ "grip", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]]
   },
   {
+    "id": "m41a_pulse_rifle",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "teal",
+    "name": "M41A pulse rifle",
+    "description": "A powerful electrolaser  developed by the R&D outfit \"Aerial Industries\".  Especially effective against electronic targets.",
+    "price": 1600000,
+    "material": ["steel", "plastic"],
+    "ammo_effects": ["DRAW_LASER_BEAM", "EMP", "BEANBAG"],
```

Comment:
Mostly because glass would stop an electrolaser form going through it, but the current induction can break it. Might be overkill, I have no qualms about using the normal "LASER" tag though.

Agreed on the beanbag and name part, will remove/change those  tomorrow.